### ISSUE TITLE
OBC.OrthoPerspectiveCamera not compatible with openbim-components 1.1.7

### ISSUE THREAD

**Issue by radoslawflis** [IFCjs/components#172]
### Describe the bug 📝

Setting instance of camera Component OrthoPerspectiveCamera for the current version of openbim-components 1.1.7 causes error.

![image](https://github.com/IFCjs/components/assets/125380891/14d09779-0e0a-47d0-9376-f8e99badf970)

The same error is visible in tutorial: https://top-docs.vercel.app/docs/Tutorials/OrthoPerspectiveCamera when you go into console

### Steps to reproduce 🔢

1. instantiate camera
`const cameraComponent = new OBC.OrthoPerspectiveCamera(viewer);
viewer.camera = cameraComponent;`
2. in package.json have open-bim components 1.1.7 installed
`  "dependencies": {
    "bootstrap": "^5.2.3",
    "openbim-components": "1.1.7",
    "three": "^0.152.2"
  }`

### System Info 💻

```shell
{
  "name": "ifc-model",
  "private": true,
  "version": "0.0.0",
  "type": "module",
  "scripts": {
    "dev": "vite",
    "build": "vite build",
    "preview": "vite preview"
  },
  "devDependencies": {
    "vite": "^5.0.0"
  },
  "dependencies": {
    "bootstrap": "^5.2.3",
    "openbim-components": "1.1.7",
    "three": "^0.152.2"
  }
}
```

### Used Package Manager 📦

npm

### Validations ✅

- [X] Read the [docs](https://platform.thatopen.com/documentation).
- [X] Check that there isn't [already an issue](https://github.com/IFCjs/components/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a repository issue and not a framework-specific issue. For example, if it's a THREE.js related bug, it should likely be reported to [mrdoob/threejs](https://github.com/mrdoob/three.js) instead.
- [X] Check that this is a concrete bug. For Q&A join our [Community](https://people.thatopen.com/).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by HoyosJuan** [comment]
Hi @radoslawflis! Just to let you know the fix was really easy, but we need to wait until is approved, merged, and probably released a new patch in 1.1.8.

**Comment by radoslawflis** [comment]
Hi @HoyosJuan Good to know it is fixed! When do you plan to relase 1.1.8?

**Comment by HoyosJuan** [comment]
Hi @radoslawflis! This issue was solved in the PR https://github.com/IFCjs/components/pull/180. I'm still doing some other things before a new patch is released to NPM. I will let you know!
